src/routes/+layout.svelte

```diff
@@ -59,7 +59,32 @@
         document.body.classList.add(className);
     }
 
-    onMount(() => {
+    onMount(async () => {
+        if (browser) {
```

Comment:
the browser check is not needed here, rather a check for dev/prod mode so we dont send stuff while developing